source/atlas-cli-deploy-local.txt

```diff
@@ -21,10 +21,6 @@ create a local |service| deployment. In this tutorial, we will deploy a
 single-node replica set on your local computer. You can then manage 
 your deployment, and use |fts| and {+avs+}.
 
-You should use local deployments for testing and development only.
-
-.. include:: /includes/fact-atlas-deployments-preview.rst
-
```

Comment:
@jvincent-mongodb can you please remove it from all 4 places `source/atlas-cli-tutorials.txt`, `source/telemetry.txt`, `source/atlas-cli-local-cloud.txt`, `source/atlas-cli-deploy-local.txt` and probably delete `source/includes/fact-atlas-deployments-preview.rst` too.
Search I used: https://github.com/search?q=repo%3Amongodb%2Fdocs-atlas-cli%20fact-atlas-deployments-preview&type=code

cc @corryroot